tokio/src/runtime/builder.rs

```diff
@@ -135,6 +135,13 @@ impl Builder {
     /// individually. If additional components are added to Tokio in the future,
     /// `enable_all` will include these future components.
     ///
+    /// # Panics
```

Comment:
Typically the "Panics" heading is for the case where `enable_all` itself panics, but this is for a different situation. 

Maybe it would be best to just move this section to the to the main `Runtime` documentation instead?

Reply:
Hello, thanks.
I move this to runtime/mod.rs in line 301. But I found line 147 has talked about Builder::enable_all and Builser::enable_time. So if this PR doesn't work, I'll close it.